lib/parser.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+// Parses protocol data unit id
+//   buffer - <Buffer>, source buffer
+//   offset - <number>, number of bytes to skip before starting to read
+// Returns: <number>, id of protocol data unit
+const parseDataUnitId = (buffer, offset) => buffer.readUInt8(offset);
```

Comment:
Isn't it ambiguous to call it `id`? Maybe `parseDataUntiType` would be better? 
WDYT @bugagashenkj @mille-nium